tests/nnstreamer_edge/nnstreamer-edge-custom-test.c

```diff
@@ -102,22 +102,38 @@ nns_edge_custom_connect (void *priv)
   nns_edge_custom_test_s *custom_h;
   nns_edge_data_h data_h;
   gchar *raw_data;
+  int ret = NNS_EDGE_ERROR_INVALID_PARAMETER;
+
   if (!priv) {
     nns_loge ("Invalid param, handle should not be null.");
-    return NNS_EDGE_ERROR_INVALID_PARAMETER;
+    return ret;
   }
   custom_h = (nns_edge_custom_test_s *) priv;
   custom_h->is_connected = 1;
 
   /* Push dummy buffer to launch GstBaseSrc */
-  nns_edge_data_create (&data_h);
+  ret = nns_edge_data_create (&data_h);
+  if (ret != NNS_EDGE_ERROR_NONE) {
+    nns_loge ("Failed to create edge data handle.");
+    return ret;
+  }
+
   raw_data = g_strdup ("Dummy data");
-  nns_edge_data_add (data_h, raw_data, strlen (raw_data) + 1, g_free);
-  nns_edge_event_invoke_callback (custom_h->event_cb, custom_h->user_data,
+  ret = nns_edge_data_add (data_h, raw_data, strlen (raw_data) + 1, g_free);
+  if (ret != NNS_EDGE_ERROR_NONE) {
+    nns_loge ("Failed to add edge data.");
```

Comment:
If failed to add data, we should release raw data `g_free (raw_data)`

Reply:
Thanks, updated.